### PR TITLE
fix repetition penalty mask in tf

### PR DESCRIPTION
**Pull request by patrickvonplaten**
Fixed bug with TF 2.0 `repetition_penalty` when doing generation add make `early_stopping` an argument to the function `generate()`.

### CODE REVIEW

**Comment by patrickvonplaten** [comment]
Good to merge for me.

File: src/transformers/configuration_utils.py [hunk 1]
```diff
@@ -70,6 +70,7 @@ def __init__(self, **kwargs):
         # Parameters for sequence generation
         self.max_length = kwargs.pop("max_length", 20)
         self.do_sample = kwargs.pop("do_sample", False)
+        self.do_sample = kwargs.pop("early_stopping", False)
```
Comment (thomwolf) [inline src/transformers/configuration_utils.py]: LHS and RHS are not the same

**Review by thomwolf — COMMENTED** [review]
Small typo bug otherwise it's good to go

**Comment by patrickvonplaten** [comment]
> Small typo bug otherwise it's good to go

Thanks for spotting!

**Pull request merged**